packages/storybook8/stories/Components/ImageOverlay/Docs.mdx

```diff
@@ -0,0 +1,31 @@
+import { Canvas, Description, Meta, Source } from '@storybook/blocks';
+
+import * as ImageOverlayStories from './index.stories';
+
+import ImageOverlayStoriesExampleText from '!!raw-loader!./snippets/ImageOverlay.snippet.tsx';
+
+<Meta of={ImageOverlayStories} />
+
+# Image Overlay
+
+Component to display image in a overlay. The overlay launches as a modal, it takes focus from the page or app and require users to interact with them.
+
+## Importing
+
+```ts
+import { ImageOverlay } from '@azure/communication-react';
+```
+
+## Example
+
+Component will render a fullscreen modal using a set image source.
+
+<Canvas of={ImageOverlayStories.ImageOverlaySnippetDocsOnly} source={{ code: ImageOverlayStoriesExampleText }} />
+
+## Cover iOS safe area
+
+To have the image overlay cover the safeArea on iOS devices, we can add `viewport-fit=cover` to the viewport meta tag of the app.
+
+```ts
+<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
```

Comment:
Do you miss the `property` table?

Reply:
![Screenshot 2024-06-28 at 3 58 02 PM](https://github.com/Azure/communication-ui-library/assets/75647512/b83273b3-20ce-4aea-ace7-4968553c77ae)

Reply:
Manually added for the time being. Need to investigate why the `<Argtypes />` tag is not working as expected